docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -97,16 +93,28 @@ This section describes optional configuration options, particularly for ensuring
 
 ## Experimental options
 
-This section describes optional configuration options which may be useful, but are not heavily-used or well-tested by the main Clearwater development team. These options should be set in the `/etc/clearwater/config` file (in the format `name=value`, e.g. `cassandra_hostname=db.example.com`).
+This section describes optional configuration options which may be useful, but are not heavily-used or well-tested by the main Clearwater development team. These options should be set in the `/etc/clearwater/shared_config` file (in the format `name=value`, e.g. `cassandra_hostname=db.example.com`).
 
 * `cassandra_hostname` - if using an external Cassandra cluster (which is a fairly uncommon configuration), a hostname that resolves to one or more Cassandra nodes.
 * `ralf_secure_listen_port` - this determines the port Ralf listens on for TLS-secured Diameter connections.
 * `hs_secure_listen_port` - this determines the port Homestead listens on for TLS-secured Diameter connections.
 * `ellis_cookie_key` - an arbitrary string that enables Ellis nodes to determine whether they should be in the same cluster. This function is not presently used.
 
+## Local Settings

```

Comment:
Should this go before shared settings?  Also should you indicate that these values live in `/etc/clearwater/local_settings`?